firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -1464,32 +1464,32 @@ extension String {
 
             public struct Options {
                 public static let LikertScaleOption1 = MZLocalizedString(
-                    key: "Microsurvey.Survey.Options.LikertScaleOption1.v127",
+                    key: "Microsurvey.Survey.Options.VerySatisfied.v131",
```

Comment:
These would be going out in v132, so they should have that, as v131 string freeze has already passed. :)

Also, this is a case where @Delphine might help out, but, given the key is getting changed, these strings will be resubmitted for translation. Should we follow our standard process where we keep the old identifiers & strings, and just add news ones?

Reply:
ah right, nice catch! I didn't realize we bumped to v132 already. Even if string freeze passes, but we didn't bump the version in main, do we still point the version after string freeze? I thought it would be depending on when the string landed. In this case, I updated since we satisfied both.

I think we did this on another PR and for that one we just changed the key name + didn't see any updates for l10n. Happy to update if that's preferred. @Delphine what are your thoughts?

Reply:
Confirmed that since this a rare occurrence and only translations for 6 strings, okay to merge. thank you!